src/components/Tabs.tsx

```diff
@@ -263,8 +263,8 @@ export function getTabStyles() {
     .pbPx(verticalPaddingPx - borderBottomWidthPx).$;
 
   return {
-    baseStyles: Css.df.aic.hPx(32).pyPx(verticalPaddingPx).px1.outline0.gray700.add("width", "fit-content")
-      .cursorPointer.sm.$,
+    baseStyles: Css.df.aic.hPx(32).pyPx(verticalPaddingPx).px1.outline0.gray700.add("width", "fit-content").overflowAuto
```

Comment:
Actually, I think we need to put that on the parent of the `role="tablist"` element. This is because we have that `{right}` property that we want within the scrollable container. We should also add a `gap` property to ensure we have some spacing between the Tab group and the `right` content.

Reply:
got it. make sense